### PR TITLE
[docs] Correcting links in DVP

### PR DESCRIPTION
**Pull request by Lada7878**
## Description

Correcting links in DVP.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

```changes
section: docs
type: chore
summary: Correcting links in DVP.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

File: docs/site/pages/virtualization-platform/documentation/admin/platform-management/node-management/ADDING_NODE_RU.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ lang: ru
 
 Чтобы добавить bare-metal сервер в кластер как статический узел, выполните следующие шаги:
 
-1. Используйте существующий или создайте новый Custom Resource [NodeGroup](../../../reference/cr.html#nodegroup) ([пример](#статические-узлы) `NodeGroup` с именем `worker`). Параметр [nodeType](../../../reference/cr.html#nodegroup-v1-spec-nodetype) в Custom Resource NodeGroup для статических узлов должен быть `Static` или `CloudStatic`.
+1. Используйте существующий или создайте новый Custom Resource [NodeGroup](../../../../reference/cr/nodegroup.html) ([пример](#статические-узлы) `NodeGroup` с именем `worker`). Параметр [nodeType](..../../../reference/cr/nodegroup.html#nodegroup-v1-spec-nodetype) в Custom Resource NodeGroup для статических узлов должен быть `Static` или `CloudStatic`.
```
Comment (z9r5) [inline docs/site/pages/virtualization-platform/documentation/admin/platform-management/node-management/ADDING_NODE_RU.md:17]:
```suggestion
1. Используйте существующий или создайте новый Custom Resource [NodeGroup](../../../../reference/cr/nodegroup.html) ([пример](#статические-узлы) `NodeGroup` с именем `worker`). Параметр [nodeType](../../../../reference/cr/nodegroup.html#nodegroup-v1-spec-nodetype) в Custom Resource NodeGroup для статических узлов должен быть `Static` или `CloudStatic`.
```

**Pull request merged**